src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs

```diff
@@ -641,7 +672,8 @@ class CheckpointFileData
         {
             public DateTime schedulingDateTime = default(DateTime);
             public long rescheduleCount = 0;
-            public bool rescheduleNeeded = false;            
+            public bool rescheduleNeeded = false;
+            public DateTime lastAttemptedUpdateTime = default(DateTime);
 
             public override string ToString()
```

Comment:
>ToString [](start = 35, length = 8)

Update ToString

Reply:
I am sorry. I forgot to push few of the local changes, I already did. I have pushed them now,